src/migrations/1671691746620-Province.ts

```diff
@@ -0,0 +1,41 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+export class Province1671691746620 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'province',
```

Comment:
```suggestion
        name: 'provinces',
```
Tên table phải số nhiều